docker/runit/init.sh

```diff
@@ -32,44 +32,5 @@ sed -i "s,<PROTOCOL>,${SPARK_SSL_PROTOCOL}," /etc/nginx/conf.d/spark.conf
 # bytes cipher strings of Java.
 # sed -i "s,<ENABLED_ALGORITHMS>,${SPARK_SSL_ENABLEDALGORITHMS//,/:}," /etc/nginx/conf.d/spark.conf
 
-# extract cert and key from keystore, write to /etc/nginx/spark.{crt,key}
-if [[ "${SPARK_SSL_ENABLED}" == true ]]; then
```

Comment:
This is no longer required because we use a similar method as the other SDK frameworks? Does this mean that the DC/OS secret store is required for SSL to work?

Reply:
@elezar This was used by the CLI before we moved to the Go CLI. Now our CLI uses the github.com/mesosphere/dcos-commons/cli/client library (same as SDK frameworks) to create an SSL connection with a cluster.